app/controllers/declaration/FiscalInformationController.scala

```diff
@@ -77,37 +76,31 @@ class FiscalInformationController @Inject()(
   private def updateCacheForYes(itemId: String, sessionId: String, updatedFiscalInformation: FiscalInformation)(
```

Comment:
You didn't change it, but this method and method for No are very similar. Could you add TODO?

`TODO Use one method instead of updateCacheForYes and updateCacheForNo` or something similar

Reply:
done